mypy/stubutil.py

```diff
@@ -90,7 +90,8 @@ def is_c_module(module: ModuleType) -> bool:
     return '__file__' not in module.__dict__ or module.__dict__['__file__'].endswith('.so')
 
 
-def write_header(file: IO[str], module_name: str, pyversion: Tuple[int, int] = (3, 5)) -> None:
+def write_header(file: IO[str], module_name: str = None,
```

Comment:
OK, I will use ``Optional[X]`` (also for previous comment) in this and next PRs. I don't have a preference concerning "syntactic sugar" just would prefer this to be consistent in function annotations and variable annotations (i.e. either both or neither).

I will implement your comments tomorrow, sorry don't have time today.